src/SignalR/src/Microsoft.AspNetCore.SignalR.Protocols.Json/baseline.netcore.json

```diff
@@ -1,253 +0,0 @@
-{
-  "AssemblyIdentity": "Microsoft.AspNetCore.SignalR.Protocols.Json, Version=1.0.0.0, Culture=neutral, PublicKeyToken=adb9793829ddae60",
```

Comment:
Are these still be verified? Should we generate a new baseline?

Reply:
I can probably copy the same baseline file and change the types manually. The methods and everything should stay the same.